README.md

```diff
@@ -28,51 +28,57 @@ on the device you specify.
 
     options:
         -v|--version        Print the LFS version this build is based on, then exit.
-        
+
         -V|--verbose        The script will output more information where applicable
                             (careful what you wish for).
-                            
+
         -e|--check          Output LFS dependency version information, then exit.
                             It is recommended that you run this before proceeding
                             with the rest of the build.
-                            
+
         -b|--build-all      Run the entire script from beginning to end.
 
+        -s|--build-iso      Run script to generate iso file. '.img' file must be 
+                            generated before executing.
+
         -x|--extend         Pass in the path to a custom build extension. See the
                             'example_extension' directory for reference.
 
         -d|--download-pkgs  Download all packages into the 'pkgs' directory, then
                             exit.
-                            
+
         -i|--init           Create the .img file, partition it, setup basic directory
                             structure, then exit.
-                            
+
+        -t|--phase-id       Specify which portion of script you need to execute. If 
+                            not specified, will run 'lfs' scripts.
+
```

Comment:
Correct me if I'm wrong, but it seems like we would only be able to build one "STARTSECTION" at a time. Is that right? I kind of like the idea of being able to build everything I need in a single command, like so:

`sudo ./mylfs.sh --build-all --extend path/to/extension`
rather than
`sudo ./mylfs.sh --build-all --phase-id lfs && sudo ./mylfs.sh --build-all --phase-id path/to/extension`

And I kind of prefer the `--extend` parameter name to the `--phase-id` parameter name. But I like where you are going with grouping phases together under a single directory.

so `my_extension/` might have:
    `phase1/`
    `phase2/`

`sudo ./mylfs.sh --build-all --extend path/to/my_extension` would build the `lfs` phases AND the `my_extension` phases all in one go. Thoughts?

Reply:
Hmm... what if make `--extend` as array? So when you add more of them they will build up in a list and build one by one...

Reply:
Then i will just edit some of `Build Extension` function to work with multiple values
And it will auto start after the building...
But then something additional option needed, cause sometimes it can broke on extension, but not on the `lfs` part

Reply:
Yeah, storing it as an array could work. And you could pass the extensions in either like `--extend path1 --extend path2...` or `--extend path1,path2,...`. Up to you.

As for restarting after a break in an extension, you could add another param like you mentioned:

```sh
# after a failure in 'my_extension'
sudo ./mylfs.sh --extend my_extension --start-extension my_extension --start-phase 2 --start-package some_package
```

Reply:
We should also make it clear that the order in which you specify extensions is important, since that will be the order in which they are built.

Reply:
ok. so i'll redo changes for `--phase-id` and extend more functionality on `--extend`

Reply:
> And you could pass the extensions in either like `--extend path1 --extend path2...` or `--extend path1,path2,...`. Up to you.

I am more prefer to have a multiple, or separated with `:` - like a `PATH` separation
So... About multiple - it will be working soon.
A oneliner - must do a separation function. But after that it will work with any symbol.